src/components/Chart/Chart.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import ClassNames from 'classnames';
+
+const Chart = ({ ...props }) => {
+  const chartType = props.data.type || 'line';
```

Comment:
I would almost rather `throw new Error(`Unsupported type=${type}`);` but that is just personal preference. When consuming downstream, if I don't see the type specified some place I will get confused...